README.md

```diff
@@ -87,6 +87,30 @@ akka.persistence {
 }
 ```
 
+### Programmatic configuration
+
+You can programmatically overrides the connection string setting in the HOCON configuration by adding a `MongoDbPersistenceSetup` to the 
+`ActorSystemSetup` during `ActorSystem` creation. The `MongoDbPersistenceSetup` takes `MongoClientSettings` instances to be used to configure
+MongoDB client connection to the server. The `connection-string` settings in the HOCON configuration will be ignored if any of these `MongoClientSettings`
+exists inside the Setup object.
+
+Example:
+```
+// Set snapshotClientSettings or journalClientSettings to null if you do not use them.
+var snapshotClientSettings = new MongoClientSettings();
+var journalClientSettings = new MongoClientSettings();
+
+// database names are not needed when its client setting is set to null
+var snapshotDatabaseName = "theSnapshotDatabase"
+var journalDatabaseName = "theJournalDatabase"
+
+var setup = BootstrapSetup.Create()
```

Comment:
Need some more examples:

1. Journal only
2. Snapshot store only
3. Both (which you have included here)

Also need to explain that you still need to set the mongodb plugin in `akka.persistence.journal` and `akka.persistence.snapshot-store` in HOCON even with these settings passed in.